### PR TITLE
- Change Likely Tournament to Valuable

### PR DESCRIPTION
**Pull request by Mebibyte**
- Add a warning to culling valuable games
- Add game name to all toast notifications
- Memoize current event for time conversion
- Change server from WEBrick to Puma
- Fix Production settings
- Make typeahead suggestions not select automatically
- Fix encoding issues in editing titles
- Improve display of game search toggles
- Fix Checked Out percentage in stats
- Improve performance of loans and storage pages
- Add page titles

Screenshots:
New search UI buttons:
screen1

Toggling Loaned out:
screen2

Trying to cull a valuable game:
screen3

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@ test:
 
 production:
   <<: *default
-  database: library_mk4_prod
+  database: library_mk4_dev
```
Comment (suzidao) [inline config/database.yml]: Did we combine dev/prod databases so there is only one? This feels like a bad idea.

File: config/secrets.yml [hunk 2]
```diff
@@ -19,4 +19,4 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: cbac2a461d1dd504876c34a879120f485c526762e341ab09840289fa5372c2ae70d87adde98eb263bd78909bf6457a4f457cbf77b7df71741228bcca7ab485d9
```
Comment (suzidao) [inline config/secrets.yml]: any particular reason for moving this out of environment?

File: app/views/admin/_game.html.erb [hunk 3]
```diff
@@ -1,8 +1,27 @@
...
+                <small>by <%= @game.by %></small>
+            </p>
+        </div>
+    </div>
+<% elsif @game %>
     <div class="row alert alert-success">
         <div class="col-xs-6" style="text-align: center; font-size: 20px;">
             <p>
-                <%= @message || 'Game successfully logged!' %>
+                <%= @message || "#{game.name} successfully logged!" %>
```
Comment (Mebibyte) [inline app/views/admin/_game.html.erb]: Should be @game.name

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>TT Library Mk5</title>
+  <title><%= content_for(:title) ? "PAXTT - #{content_for(:title)}" : "TT Library Mk5" %></title>
```
Comment (Mebibyte) [inline app/views/layouts/application.html.erb]: Change PAXTT to TTLib, for non-PAX shows.

File: config/database.yml (hunk 1 above)
Comment (Mebibyte) [reply config/database.yml]: We've been running in dev mode exclusively. Switching to Prod mode now for better performance. I'll copy everything from the dev DB to prod and change this back to prod DB! Thanks!

File: config/secrets.yml (hunk 2 above)
Comment (Mebibyte) [reply config/secrets.yml]: Just copied the one from dev to prod. Will change back to using prod's key in Env! Thanks!

**Pull request merged**